csharpcore/GildedRoseTests/GildedRoseTest.cs

```diff
@@ -1,18 +1,65 @@
-﻿using Xunit;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using GildedRoseKata;
+using NUnit.Framework;
 
 namespace GildedRoseTests
 {
+    [TestFixture]
     public class GildedRoseTest
     {
-        [Fact]
-        public void foo()
+        [TestFixture]
+        public class UpdateQuality
         {
-            IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
-            GildedRose app = new GildedRose(Items);
-            app.UpdateQuality();
-            Assert.Equal("fixme", Items[0].Name);
+            [TestFixture]
+            public class ForEachStandardItem
+            {
+                [Test]
+                public void ShouldDecreaseSellInByOne()
+                {
+                    // Arrange
+                    var standardItem = new Item { Name = "Standard item", SellIn = 10, Quality = 10 };
+                    IList<Item> items = new List<Item> { standardItem };
+
+                    // Act
+                    new GildedRose(items).UpdateQuality();
+
+                    // Assert
+                    Assert.That(standardItem.SellIn, Is.EqualTo(9));
+                }
+
+                [TestFixture]
+                public class ShouldDecreaseQuality
+                {
+                    [Test]
+                    public void ByOneBeforeSellInDate()
+                    {
+                        // Arrange

```

Comment:
I add the steps of the test in comment so you can follow them but if you know them we can remove them, that's pollution